requirements.txt

```diff
@@ -2,7 +2,6 @@ numpy
 scipy
 torchio
 torch
-logging
```

Comment:
What was the problem here? I'm not sure where it's used but it also seems strange that it would cause issues.

Reply:
Yeah, logging is now part of python standard library.